### PR TITLE
config: Replace log outputs with fmt.Fprintln.

### LINKED ISSUE #757: Remove btcdLog.Warn before logging is initialized

### PR DESCRIPTION
**Pull request by marcopeereboom**
Fixes #757.

### CODE REVIEW

**Comment by davecgh** [comment]
Nice catch.  OK.

File: config.go [hunk 1]
```diff
@@ -833,8 +834,8 @@ func loadConfig() (*config, []string, error) {
 
 		if cfg.TorIsolation &&
 			(cfg.ProxyUser != "" || cfg.ProxyPass != "") {
-			btcdLog.Warn("Tor isolation set -- overriding " +
-				"specified proxy user credentials")
+			fmt.Fprintln(os.Stderr, "Tor isolation set -- "+
```
Comment (davecgh) [inline config.go]: It's safe to use logging at this point since it has been initialized.  Any reason to switch it?

File: config.go [hunk 2]
```diff
@@ -871,8 +872,9 @@ func loadConfig() (*config, []string, error) {
 
 		if cfg.TorIsolation &&
 			(cfg.OnionProxyUser != "" || cfg.OnionProxyPass != "") {
-			btcdLog.Warn("Tor isolation set -- overriding " +
-				"specified onionproxy user credentials ")
+			fmt.Fprintln(os.Stderr, "Tor isolation set -- "+
```
Comment (davecgh) [inline config.go]: Same as previous.

File: config.go (hunk 1 above)
Comment (marcopeereboom) [inline config.go]: That is correct, however, people tend to copy paste and now by having everything identical there is no excuse to do it wrong.  Larting can then ensue appropriately.

File: config.go (hunk 2 above)
Comment (marcopeereboom) [inline config.go]: Ditto.

File: config.go (hunk 1 above)
Comment (davecgh) [inline config.go]: Fair enough.  I have no objection to that.

**Comment by davecgh** [comment]
OK

**Pull request merged**